packages/grid/x-data-grid/src/components/panel/filterPanel/GridFilterForm.tsx

```diff
@@ -476,6 +498,7 @@ const GridFilterForm = React.forwardRef<HTMLDivElement, GridFilterFormProps>(
               applyValue={applyFilterChanges}
               focusElementRef={valueRef}
               {...currentOperator.InputComponentProps}
+              {...valueInputTextFieldProps}
```

Comment:
👍That was another option I had in mind

About the naming, what do you think of `TextFieldProps` instead of `InputComponentProps` to be more specific?

Reply:
We can't know if the input component is a text field or not, it could be anything. The filter operator already has `InputComponentProps`. I was considering this name for consistency.

Reply:
But `variant` is not part of the `InputComponentProps`.
The InputComponent will be a `FilledInput`, `OutlinedInput` or `Input` component depending on the `variant` prop value

Reply:
My idea was to allow any prop in `InputComponentProps`, not restricting to only those from the text  field variants. It's the role of the current input component to use the `variant` received.

As example, the input component of the operators of a `singleSelect` column is not a TextField, but an Autocomplete. We need to forward the prop to the element that can use it.

```diff
diff --git a/packages/grid/x-data-grid/src/components/panel/filterPanel/GridFilterInputMultipleSingleSelect.tsx b/packages/grid/x-data-grid/src/components/panel/filterPanel/GridFilterInputMultipleSingleSelect.tsx
index e757992be..d3fa254ca 100644
--- a/packages/grid/x-data-grid/src/components/panel/filterPanel/GridFilterInputMultipleSingleSelect.tsx
+++ b/packages/grid/x-data-grid/src/components/panel/filterPanel/GridFilterInputMultipleSingleSelect.tsx
@@ -24,7 +24,7 @@ const isOptionEqualToValue: GridFilterInputMultipleSingleSelectProps['isOptionEq
 const filter = createFilterOptions<any>();
 
 function GridFilterInputMultipleSingleSelect(props: GridFilterInputMultipleSingleSelectProps) {
-  const { item, applyValue, type, apiRef, focusElementRef, ...other } = props;
+  const { item, applyValue, type, apiRef, focusElementRef, variant = 'standard', ...other } = props;
   const id = useId();
 
   const resolvedColumn = item.columnField ? apiRef.current.getColumn(item.columnField) : null;
@@ -118,7 +118,7 @@ function GridFilterInputMultipleSingleSelect(props: GridFilterInputMultipleSingl
           }}
           inputRef={focusElementRef}
           type={'singleSelect'}
-          variant="standard"
+          variant={variant}
         />
       )}
       {...other}
```